src/modules/site-v2/templates/tools/phenotype/submit.html

```diff
@@ -0,0 +1,71 @@
+{% extends "_layouts/default-nocontainer.html" %}
+
+
+{% block content %}
+  {% from "_includes/macros.html" import render_field %}
+
+  <div class="container-fluid px-5">
+    <form id='form-submit' enctype="multipart/form-data" method="POST">
+      {{ form.csrf_token }}
+      <input hidden name="jwt_csrf_token" value="{{ jwt_csrf_token }}">
+      <div class='mb-4'>{{ render_field(form.species) }}</div>
+      <div class='mb-4'>{{ render_field(form.label, placeholder="Name your calculation.") }}</div>
+      <div class="mb-4">{{ render_field(form.trait_1) }}</div>
+      <div class="mb-4">{{ render_field(form.trait_2) }}</div>
+      <div class="d-grid">
+          <button type="submit" id="submit-btn" class="btn btn-secondary text-light" aria-label="submit">Submit</button>
+      </div>
+    </form>
+  </div>
+
+{% endblock %}
+
+
+{% block script %}
+{% from "_includes/strain-listing.html" import unpack_species_list  %}
+{% from "_scripts/submit-job.js"        import def_submit_job       %}
+
+<script>
+  {% include '_scripts/utils.js' %} {#/* defines: flash_message */#}
+  {{ def_submit_job('phenotype_comparison', true) }}
+
+
+  /* Species */
+  const species_list = {{ unpack_species_list(species_list, species_fields) }};
+  let species;
```

Comment:
not used?

Reply:
Not yet -- we'll want to do some species verification based on the selected trait(s).  I'll have a better idea how to do that once we figure out the trait table.